### PR TITLE
Apply automatic fixes (manually)

### PR DESCRIPTION
**Pull request by andschwa**
These were available in code lens style "click to fix" but were not
applied by the CLI tools. It was slow, but not horrible.

### CODE REVIEW

File: src/PowerShellEditorServices.Hosting/Internal/NamedPipeUtils.cs [hunk 1]
```diff
@@ -111,8 +109,7 @@ public static string GenerateValidNamedPipeName(IReadOnlyCollection<string> pref
                 {
                     return pipeName;
                 }
-
-            } while (tries < 10);
```
Comment (andschwa) [inline src/PowerShellEditorServices.Hosting/Internal/NamedPipeUtils.cs:115]: This was never incrementing tries and therefore had actually been an infinite loop.

File: src/PowerShellEditorServices/Hosting/EditorServicesServerFactory.cs [hunk 2]
```diff
@@ -132,15 +124,13 @@ private EditorServicesServerFactory(ILoggerFactory loggerFactory, LogLevel minim
         public PsesDebugServer RecreateDebugServer(
             Stream inputStream,
             Stream outputStream,
-            PsesDebugServer debugServer,
-            bool usePSReadLine)
```
Comment (andschwa) [inline src/PowerShellEditorServices/Hosting/EditorServicesServerFactory.cs:136]: Apparently we don't use `usePSReadLine` at all.

**Comment by andschwa** [comment]
<200 "problems" now!

File: src/PowerShellEditorServices/Services/DebugAdapter/Handlers/BreakpointHandlers.cs [hunk 3]
```diff
@@ -128,8 +128,7 @@ public async Task<SetFunctionBreakpointsResponse> Handle(SetFunctionBreakpointsA
             CommandBreakpointDetails[] breakpointDetails = request.Breakpoints
                 .Select((funcBreakpoint) => CommandBreakpointDetails.Create(
                     funcBreakpoint.Name,
-                    funcBreakpoint.Condition,
-                    funcBreakpoint.HitCondition))
```
Comment (andschwa) [inline src/PowerShellEditorServices/Services/DebugAdapter/Handlers/BreakpointHandlers.cs:132]: Last param was unused in the function.

File: src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs [hunk 4]
```diff
@@ -368,21 +368,18 @@ public async Task<string> SetVariableAsync(int variableContainerReferenceId, str
...
 
-            // OK, now we have a PS object from the supplied value string (expression) to assign to a variable.
-            // Get the variable referenced by variableContainerReferenceId and variable name.
-            VariableContainerDetails variableContainer = null;
             await debugInfoHandle.WaitAsync().ConfigureAwait(false);
             try
             {
-                variableContainer = (VariableContainerDetails)variables[variableContainerReferenceId];
+                // OK, now we have a PS object from the supplied value string (expression) to assign to a variable.
+                // Get the variable referenced by variableContainerReferenceId and variable name.
```
Comment (andschwa) [inline src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs:375]: Sorry this file might conflict with your PR.

**Comment by andschwa** [comment]
> <200 "problems" now!

And now only 75!

**Review by SeeminglyScience — APPROVED** [review]
LGTM!

**Pull request merged**